mock/cpp_mock_scenarios/CMakeLists.txt

```diff
@@ -53,6 +53,8 @@ if(BUILD_CPP_MOCK_SCENARIOS)
   add_subdirectory(src/pedestrian)
   add_subdirectory(src/random_scenario)
   add_subdirectory(src/speed_planning)
+  # add_subdirectory(src/respawn_ego)
```

Comment:
This line is commented because otherwise CI build step was not successful due to an error: https://github.com/tier4/scenario_simulator_v2/actions/runs/8005447800/job/21864899790#step:13:136.

I think this might be the general error related to cpp_mock_scenarios which use Autoware. Commenting this line and excluding the `respawn_ego` scenario from the build is the temporary solution making the CI pass.

Reply:
Whether it is by OpenSCENARIO or a scenario written directly in C++, a build of Autoware is required to run a scenario containing Autoware on GitHub Actions.
In order to avoid too long execution times for GitHub Actions, and because the success of a scenario cannot be guaranteed since Autoware's behavior is not definitive, the repository is operated under the policy that tests involving Autoware are not performed on GitHub Actions.